sktime/classification/shapelet_based/_rocket_classifier.py

```diff
@@ -102,31 +110,35 @@ def fit(self, X, y):
         self : object
         """
         X, y = check_X_y(X, y)
+        n_jobs = check_n_jobs(self.n_jobs)
 
         self.n_classes = np.unique(y).shape[0]
         self.classes_ = class_distribution(np.asarray(y).reshape(-1, 1))[0][0]
         for index, classVal in enumerate(self.classes_):
             self.class_dictionary[classVal] = index
 
         if self.ensemble:
-            for i in range(self.ensemble_size):
-                rocket_pipeline = make_pipeline(
-                    Rocket(
-                        num_kernels=self.num_kernels, random_state=self.random_state
-                    ),
-                    RidgeClassifierCV(alphas=np.logspace(-3, 3, 10), normalize=True),
+            if n_jobs is None:
+                for _ in range(self.ensemble_size):
+                    base_estimator = _make_estimator(
+                        self.num_kernels, self.random_state
+                    )
+                    self.classifiers.append(_fit_estimator(base_estimator, X, y))
+            else:
+                base_estimator = _make_estimator(self.num_kernels, self.random_state)
+                self.classifiers = Parallel(n_jobs=n_jobs)(
```

Comment:
`self.classifiers` should be `self.estimators_` with the trailing underscore indicating fitted parameters

Reply:
Same comment about compatibility after renaming. Should be less of an issue in this case though `classifiers` is mentioned in docs as a public attribute of the `ROCKETClassifier` class. Which means it could be used outside of the scope of the library.

Reply:
Yes, see above comment